app/helpers/admin/audit_trail_helper.rb

```diff
@@ -19,7 +19,11 @@ def describe_audit_trail_entry(entry)
   end
 
   def paginated_audit_trail_url(page)
-    url_for(params.permit!.except(:host, :controller, :action).merge(controller: "admin/edition_audit_trail", action: "index", page: (page <= 1 ? nil : page)))
+    url_for(
+      params.slice(:id, :page)
+            .permit!
+            .merge(controller: "admin/edition_audit_trail", action: "index", page: (page <= 1 ? nil : page)),
+    )
```

Comment:
Does this risk dropping some parameters? I wonder if brakeman will be happy if you do `params.to_unsafe_hash.merge(controller: "admin/edition_audit_trail", action: "index", page: (page <= 1 ? nil : page))` here?  might also need a symbolize_keys for a nice merge.

Reply:
Okay done - Brakeman doesn't have any qualms with to_unsafe_hash